Device.js

```diff
@@ -1,55 +1,31 @@
+var { NativeModules } = require('react-native');
```

Comment:
'destructuring expression' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'require' is not defined.